src/app/constants/helpers.js

```diff
@@ -1,5 +1,47 @@
+// TODO move this to a utils file since these aren't constants.
+import pick from 'lodash/pick';
+
 import { API_DOMAIN } from './paths';
 
 export function apiUrl(path) {
   return `${API_DOMAIN}/${path}`;
 };
+
+export function getRsvpGroupUrl(code) {
+  return apiUrl(`rsvp/show/${code}`);
+};
+
+export function postRsvpGroupUrl(userGroupId) {
+  return apiUrl(`rsvp/${userGroupId}`);
+};
+
+const VALID_USER_GROUP_FIELDS = [
```

Comment:
@lxwyndxl we don't want users to be able to edit everything (like name since we are not allowing rando +1s). This whitelists editable fields.